blog/drafts-unpublished/announcing-rome/README.md

```diff
@@ -0,0 +1,90 @@
+# Post
+
+#### Title
+
+Announcing Rome
+
+#### Purpose
+
+Announcing our next testnet, Rome
+
+#### url
+
+blog.joystream.org/announcing-rome/
+blog.joystream.org/p/715106e8-6671-490b-aea1-cb5754cc524b/
+
+#### Cover
+
+<p align="center"><img src="announcing-rome-cover.png"></p>
+
+#### Lead
+
+After launching the Acropolis network in late June, we are finally ready to announce the plans for our next testnet - Rome.
+
+#### Body
+
+---
+
+## What is Rome
+
+The Rome network will be revolve around one key platform feature - a flexible, adaptable and user friendly Content Directory. Our reasoning for focusing on this task, that may seem narrow and trivial at first glance, is the long term value of it:
```

Comment:
No need to capitalise content directory?

Reply:
Maybe not...